doc/sections/structure.md

```diff
@@ -13,6 +13,17 @@ Structure of directories and files
    + `vim/`: code for vim
    + `vscode/`: code for vscode
 
+     - `.vscode/*.json`: config for launching, debugging, building the extension
+     - `src/*.ts`: the VSCode plugin source code written in the TypeScript language
+     - `package.json`: the manifest of the plugin (activation events, scripts, dependencies, ...)
+     - `tsconfig.json`: TypeScript configuration (directories, ...)
+     - `vscode.proposed.d.ts`: unstable VSCode API (likely to change)
+     - `lp.configuration.json`: specific characters
+     - `syntaxes/lp.tmLanguage.json`: grammar of lambdapi [WIP : currently only all the reserved keywords are highlighted in one unique color]
+     - `Makefile`
+     - `.gitignore`
+     - `License-vscoq.txt` MIT License mentionning the original author of VSCoq, a similar plugin for Coq
```

Comment:
Why do we need this License file? I would prefer that the content of this file is included in the file(s) taken from vscoq.

Reply:
I let @ejgallego and @rlepigre comment on this one
(I think that's pretty standard to include a License file like this)

Reply:
That's normal indeed to include the License text in the files taken from Vscoq but putting a license file in this directory may suggest that all files are taken from Vscoq which is wrong. That's why it is important to identify that files that are coming from Vscoq.

Reply:
Hello @ejgallego . Could you please tell us what are the files that have been taken from VSCoq?

Reply:
Originally the whole plugin was taken, but at this point not much remains.

Reply:
@ejgallego Do we really need to keep this license file then? If you want, we can add a THANKS.md file where we say that you took some inspiration from VSCoq instead.

Reply:
I guess you can remove it if you want and indeed place a note on the main `LICENSE` or`THANKS` file; licensing stuff is always complicated and has not definitive answer. Usually it doesn't bother me to keep extra stuff.

But after a review it is safe to say that no original code does remain tho.